docs/intro_04_interfaces.md

```diff
@@ -1,19 +1,19 @@
 ---
-id: introduction6
-title: How one can make use of the Gnosis Safe Contracts?
-sidebar_label: How one can make use of the Gnosis Safe Contracts?
+id: intro_interfaces
+title: Gnosis Safe Interfaces
+sidebar_label: Gnosis Safe Interfaces
 ---
 
-Utilization of the core Gnosis Safe features are already implemented within our own products. Some of these examples extended functionalities are as follows.
+The Gnosis Safe smart contracts are used in many different interfaces. Gnosis is developing multiple different of these:
```

Comment:
The second sentence has a grammar bug. I would remove the second one and write for the first one:

`The Gnosis Safe smart contracts can be access via different interfaces:`